### ISSUE TITLE
GPIO misunderstanding

### ISSUE THREAD

**Issue by shirish47** [PetteriAimonen/ED060SC4_driver#1]
In file gdisp_lib_board_example.h

According to sch.
 in line #74 you have used GPIOA but your Schematic shows GPIOC connected to EINK_VDD and you have not defined GPIOA_EINK_VDD but instead you have defiend GPIOB_EINK_VDD which is not according to Sch. and also same for PORTC pins.. is it your mistake or something . Kindly pls let us know. :) 
Thanks

**Comment by PetteriAimonen** [comment]
Yeah, probably my mistake. I changed the schematic a bit from what it 
was for my own proto pcb, maybe I forgot to change some pin mappings in 
the code.
